packages/app-project/stores/Recents.spec.js

```diff
@@ -0,0 +1,139 @@
+import { expect } from 'chai'
+import sinon from 'sinon'
+import { getSnapshot } from 'mobx-state-tree'
+import asyncStates from '@zooniverse/async-states'
+import { panoptes } from '@zooniverse/panoptes-js'
+
+import Project from './Project'
+import Store from './Store'
+import initStore from './initStore'
+import placeholderEnv from './helpers/placeholderEnv'
+
+describe('Stores > Recents', function () {
+  let rootStore
+  const project = {
+    id: '2',
+    display_name: 'Hello',
+    slug: 'test/project'
+  }
+
+  before(function () {
+    const mockResponse = {
+      body: {
+        recents: [
+          {
+            locations: [
+              { 'image/jpeg': 'subject.jpeg'}
+            ],
+            links: {
+              subject: '345'
+            }
+          }
+        ]
+      }
+    }
+    rootStore = initStore(true, { project })
+    sinon.stub(rootStore.client.panoptes, 'get').callsFake(() => Promise.resolve(mockResponse))
+  })
+
+  after(function () {
+    rootStore.client.panoptes.get.restore()
+  })
+
+  it('should exist', function () {
+    expect(rootStore.recents).to.be.ok
+  })
+
+  describe('with a project and user', function () {
+    before(function () {
+      const user = {
+        id: '123',
+        login: 'test.user'
+      }
+      sinon.stub(rootStore.collections, 'fetchFavourites')
```

Comment:
This is stubbed so that it doesn't call `client.panoptes.get`. It might be cleaner to change the `panoptes.get` tests to expect `.called` rather than `.calledOnce`. Otherwise, I can see this block having to be updated every time a new API call is added to a store.

Reply:
The API would also need to be mocked so that calls like this don't error when the user store updates.

Reply:
I've run into the same problem with mocking the get requests in the classifier store tests. We should make notes about this and try to come up with a good consistent way to mock the panoptes.js client. 

Checking the number of times something is called though is something that I've found has helped me and actually I caught an issue in the Tutorial store because an action was being called twice when it shouldn't be. So ideally, maybe we have the client mocked up for all the tests to use and use a default afterEach to reset its history?

Reply:
In #483 I've been stubbing methods on the default API client but that can be cumbersome too.

I'd really like this to check that the API is only called once, for the same reasons. The problem I ran into here was how to ignore all the other API requests that happen during store initialisation. When my `calledOnce` test ran, it failed because `client.panoptes.get` had been called twice. Resetting the history wouldn't fix that, but better mocking of the client might.